package.json

```diff
@@ -69,6 +70,7 @@
     "cross-env": "^7.0.2",
     "delay-cli": "^1.1.0",
     "faucet": "0.0.1",
+    "husky": "^4.2.5",
```

Comment:
Since `husky` has been regarded as a footgun for developers by some of our contributors – and seeing that it has been added and [removed](https://github.com/nodejs/nodejs.dev/pull/733) from this repo multiple times due to that friction, elevating the conversation about how we want to handle commit-linting across the project (as @MylesBorins mentioned [here](https://github.com/nodejs/nodejs.dev/pull/733#issuecomment-631634322)) seems like the right thing to do – rather than reintroducing it here just to help the i18n initiative protect one repo that consumes its exports from commits made directly to it (and unintentionally green-light it for other cases in `/nodejs.org`). Imo, I agree that when there is project-wide consensus on how we want to enforce commit linting across (eg. via `husky` or as a CI job)  – then we can be more confident on how we want 'source of truth' repos like `/i18n` to handle this in order to prevent direct commits elsewhere. 👍

Given that, I'm going to request that we at the very least:
* Hear more opinions here about approving this PR which reintroduces `husky` into this repo, and hopefully get greater consensus on that beyond the folks who're involved with `i18n` and `nodejs.org`.
* Elevate the conversation accordingly to the TSC via [admin](https://github.com/nodejs/admin).

cc @alexandrtovmach @zeke @MylesBorins

Reply:
cc @nodejs/website

Reply:
@zeke, I think for linting js files, [standard](https://standardjs.com/) is being used. As shown in this snippets, https://github.com/nodejs/nodejs.org/blob/21353d0db79a70b207cdb24b114affaa7b807e12/package.json#L15-L24

If its requires to `npm run test` before each commit, then standard should lint the js file while other modules are used in parallel to lint different type of files  as shown above.